tests/runtime/basic

```diff
@@ -96,6 +96,12 @@ EXPECT kretfunc
 REQUIRES_FEATURE btf
 TIMEOUT 1
 
+NAME listing with wildcarded probe type
```

Comment:
Just to make sure, these newly added tests are worked as the same as before? or some behavior are changed?

Reply:
This one works the same as before, but e.g. listing userspace probes by wildcarded probe type didn't work.